### DISCUSSION TITLE
Topic status not saved for repo

### DISCUSSION THREAD

**Discussion by thomasheartman** [magit/forge#366] [Support] [answered]
One of the repos I work with regularly seems to not store its forge/topic data in between reboots. Is this a configuration setting I've hit by mistake somehow or is this unintended?

From what I can tell, other repos store the data so that the 'issues' and 'pull requests' tabs are available on the initial Magit status page. For this one repo, however, it only shows the basic Magit status page when I open it for the first time after a reboot. To get forge data, I need to explicitly fetch forge data again.

I'm happy to provide any additional info you want. I haven't tried with the latest releases just yet, but will shortly. If that resolves it, I'll report back 🙋🏾 

Thanks! 🙏🏾

**Comment by tarsius** [comment, marked as answer]
Check if the name of the owner and repository is exactly the same everywhere. Including case. Check specifically the url of the git remote.

**Reply by thomasheartman** [reply]
Thanks! What are all the places it can be listed? The git remote is all lowercase (copied from github), and I don't think I've treated this repo any differently than any other repo. Are there any other places I should check?

**Reply by tarsius** [reply]
Na, the places you already checked is all that I meant.

**Reply by thomasheartman** [reply]
Aight, cool. The repo in question was renamed yesterday (after I reported the issue), and now I'm not seeing any issues, even when I pull. I tried deleting my local repo and re-cloning it. I added it to the database and now when I fetch all topics, all I'm getting is 'Running git fetch origin' and 'Git finished'. Nothing happens in the magit buffer. Is this related, you think? 

I can't remember experiencing this with any other repos either 🤔

**Reply by tarsius** [reply]
It's not easy for me to know from a distance what might be going on and what inconsistent state things are in by now. Yes, I do think that all these things are related and that the initial cause is the name change. I also think that if you deleted all data and started from scratch, that things would probably work then. Please try that. (Start with a fresh repo *and* database.)

**Reply by thomasheartman** [reply]
Yeah, that's a good point. I didn't realize you could delete the database 😅  But I tried that out now, and at least pulling topics worked again 🎉  I'll try to start with a new repo as well if it doesn't save the state between reboots again.

Thanks for the input! 🙏🏾
